content/en/blog/_posts/2022-01-19-Securing-Admission-Controllers.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: blog
+title: "Securing Admission Controllers"
+date: 2022-01-19
+slug: secure-your-admission-controllers-and-webhooks
+---
+
+[Admission control](/docs/reference/access-authn-authz/admission-controllers/) is a key part of Kubernetes security, alongside authentication and authorization. Webhook admission controllers are extensively used to help improve the security of Kubernetes clusters in a variety of ways including restricting the privileges of workloads and ensuring that images deployed to the cluster meet organization’s security requirements.
+
+However, as with any additional component added to a cluster, there are security risks which can present themselves, if the deployment and management of the admission controller isn’t handled correctly.  To help admission controller users and designers manage these risks appropriate, the [security documentation](https://github.com/kubernetes/community/tree/master/sig-security#security-docs) subgroup of SIG Security have spent some time developing a [threat model for admission controllers](https://github.com/kubernetes/sig-security/tree/main/sig-security-docs/papers/admission-control). This threat model looks at likely risks which may arise from the incorrect use of admission controllers, which could allow security policies to be bypassed, or even allow an attacker to get unauthorised access to the cluster.
+
+From the threat model, we developed a set of security best practices which should be adopted to ensure that cluster operators can get the security benefits of admission controllers whilst avoiding any risks from using them.
+
+## Admission controllers and good practices for security
+
+From the threat model there were a couple of themes which emerged around how to ensure the security of admission controllers.
```

Comment:
```suggestion
From the threat model, there were a couple of themes that emerged around how to ensure the security of admission controllers.
```

Reply:
we changed the wording on this one, so I think it's redundant.